app/src/main/java/com/lagradost/cloudstream3/ui/result/ResultFragmentPhone.kt

```diff
@@ -700,6 +701,11 @@ open class ResultFragmentPhone : FullScreenPlayer() {
                     resultCastItems.isGone = d.actors.isNullOrEmpty()
                     (resultCastItems.adapter as? ActorAdaptor)?.updateList(d.actors ?: emptyList())
 
+                    if (d.contentRatingText == null) {
+                        // If there is no rating to display, we don't want an empty gap
+                        resultMetaContentRating.width = 0
```

Comment:
:raised_eyebrow: what if the contentRatingText gets populated after it was null, does it work then?

Reply:
> 🤨 what if the contentRatingText gets populated after it was null, does it work then?

It shouldn't be populated after it is null, Ive never encountered it happening anyway, but we can add something to ensure that I suppose if needed?

Reply:
If it is null here, then it would be null a few lines earlier when it does `resultMetaContentRating.setText(d.contentRatingText)` so it wouldn't display anything anyways and this should trigger removing the width was my understanding anyway.